interface.rb

```diff
@@ -0,0 +1,277 @@
+class Interface
+
+  attr_accessor :trains, :stations, :routes, :wagons
+
+  def initialize
+    @trains = []
+    @routes = []
+    @stations = []
+    @wagons=[]
+  end
+#-------------------------------------------------------------------------------iterator
+=begin 
+def iterator(arr)                                     не понимаю, почему оно не работает
+    arr.each.with_index(1) do |value, index|
+      puts "#{index}: #{value.to_s}"
+    end
+    arr[gets.to_i]
```

Comment:
```suggestion
    arr[gets.to_i - 1]
```
только внутри блока итератора у тебя индекс (локальная переменная блока с именем "индекс") с 1 начинается, индексирование элементов в самом массиве начинается все также с 0, массив свои свойства не меняет